### PR TITLE
Fix hwintrinsicChild->isContained() assert

### LINKED ISSUE #90508: Assertion failed 'hwintrinsicChild->isContained()'
```cs
using System.Runtime.CompilerServices;
using System.Runtime.Intrinsics;

public class Tests
{
    public static void Main()
    {
        Div(default, 1);
    }

    [MethodImpl(MethodImplOptions.NoInlining)]
    static Vector128<double> Div(Vector128<double> v, double b) => v / b;
}
```
Fails on Checked, win-x64, Tier0 with:
```
Assert failure(PID 48468 [0x0000bd54], Thread: 44484 [0xadc4]): Assertion failed 'hwintrinsicChild->isContained()' in 'mytest:Foo(System.Runtime.Intrinsics.Vector128`1[double],double):System.Runtime.Intrinsics.Vector128`1[double]' during 'Generate code' (IL size 8; hash 0xece1a119; Tier0)

    File: C:\prj\runtime-main\src\coreclr\jit\instr.cpp Line: 858
    Image: C:\prj\runtime-main\artifacts\bin\coreclr\windows.x64.Checked\CoreRun.exe
```

### PR DESCRIPTION
**Pull request by EgorBo**
Fixes https://github.com/dotnet/runtime/issues/90508

### CODE REVIEW

**Comment by kunalspathak** [comment]
are you still working on this @EgorBo ?

**Comment by BruceForstall** [comment]
> are you still working on this @EgorBo ?

If not, please make it "Draft"

**Comment by EgorBo** [comment]
Oops, didn't get notifications for this one -- yes, I'll finish it this week, fixes https://github.com/dotnet/runtime/issues/90508

File: src/tests/JIT/Regression/JitBlue/Runtime_90508/Runtime_90508.cs [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        v + Sse3.MoveAndDuplicate(Vector128.CreateScalarUnsafe(1.0));
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static Vector128<double> Div3(Vector128<double> v, double b) => 
+        v + Sse3.MoveAndDuplicate(Vector128.Create(1.0));
+
+    [Fact]
+    public static int TestEntryPoint()
+    {
+        if (!Ssse3.IsSupported)
```
Comment (BruceForstall) [inline src/tests/JIT/Regression/JitBlue/Runtime_90508/Runtime_90508.cs:26]: Is there one too many `s` here?
```suggestion
        if (!Sse3.IsSupported)
```

Looks like the APIs being used are all `Sse3`?

File: src/tests/JIT/Regression/JitBlue/Runtime_90508/Runtime_90508.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+using System.Runtime.CompilerServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.X86;
+using Xunit;
+
+public class Runtime_90508
+{
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static Vector128<double> Div1(Vector128<double> v, double b) =>
```
Comment (BruceForstall) [inline src/tests/JIT/Regression/JitBlue/Runtime_90508/Runtime_90508.cs:12]: nit: why are these functions named `DivX`?

Comment (EgorBo) [reply src/tests/JIT/Regression/JitBlue/Runtime_90508/Runtime_90508.cs]: Oops, no idea 🙂 changed to Test

**Comment by JulieLeeMSFT** [comment]
@EgorBo, I reran the failed tests and it alls passed. Is it ready to merge?

**Comment by EgorBo** [comment]
Should be ready now, waiting for spmi diffs

File: src/tests/JIT/Regression/JitBlue/Runtime_90508/Runtime_90508.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+using Xunit;
+
+public class Runtime_90508
+{
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static Vector128<double> Test1(Vector128<double> v, double b) =>
+        v + Sse3.MoveAndDuplicate(Vector128.CreateScalarUnsafe(b));
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static Vector128<double> Test2(Vector128<double> v, double b) =>
```
Comment (kunalspathak) [inline src/tests/JIT/Regression/JitBlue/Runtime_90508/Runtime_90508.cs:16]: nit: `b` in `Test2` and `Test3` are unused.

**Review by kunalspathak — APPROVED** [review]
LGTM

**Pull request merged**